CONTRIBUTING.md

```diff
@@ -1,303 +1,61 @@
 
-[`IPFS Main Repo`](https://github.com/ipfs/ipfs#ipfs---the-permanent-web) • [`IPFS Project Directory`](https://github.com/ipfs/ipfs#project-directory)
 
-Want to hack on IPFS? Awesome! Here are instructions to get you started. They are not perfect yet. Please let us know what feels wrong or incomplete.
 
-IPFS is an Open Source project and we welcome contributions of all sorts. There are many ways to help, from reporting issues, contributing code, and helping us improve our community.
+Moving to a more distributed web is a big task and we need contributors of all skill levels and backgrounds.
 
-## Topics
 
-- [Security Issues](#security-issues)
-- [Community Guidelines](#community-guidelines)
-- [Dive Right In](#dive-right-in)
-- [Reporting Issues](#reporting-issues)
-- [Protocol Design](#protocol-design)
-- [Implementation Design](#implementation-design)
-- [Community Improvement](#community-improvement)
-- [Translations](#translations)
-- [Helping in other ways](#helping-in-other-ways)
-- [Code Contribution Guidelines](#code-contribution-guidelines)
-  - [Discuss big changes as Issues first](#discuss-big-changes-as-issues-first)
-  - [Pull Requests always welcome](#pull-requests-always-welcome)
-  - [Conventions](#conventions)
-    - [Go Code Contributing Guidelines](#go)
-    - [JavaScript  Code Contributing Guidelines](#javascript)
-    - [Git](#git)
-    - [Commit messages](#commit-messages)
-      - [Subject line should not be more than 80 characters long](#subject-line-should-not-be-more-than-80-characters-long)
-      - [A "License" and a "Signed-off-by" trailers are required](#a-license-and-a-signed-off-by-trailers-are-required)
-      - [Commit message examples](#commit-message-examples)
-    - [Code](#code)
-    - [Tests](#tests)
-    - [Documentation](#documentation)
-    - [Pull Requests](#pull-requests)
-    - [Code Review](#code-review)
-    - [Rebasing](#rebasing)
-    - [Merge Approval](#merge-approval)
-- [Credits](#credits)
+Below are instructions on how to contribute to the IPFS project. What does it mean to contribute? Contributing means that you are helping with the coding, design, or writing the documentation of IPFS. 
 
 
-## Security Issues
+### ⏰ Abbreviated Instructions on How to Submit a Pull Request
 
-The IPFS protocol and its implementations are still in heavy development. This means that there may be problems in our protocols, or there may be mistakes in our implementations. And -- though IPFS is not production-ready yet -- many people are already running nodes in their machines. So we take security vulnerabilities very seriously. If you discover a security issue, please bring it to our attention right away!
+* [ ] A little bit shy? Report a bug.
```

Comment:
Could we link "Report a bug" to the appropriate issue template? Or is this documentation for people who'll work across a variety of different repos with different issue queues?